README.md

```diff
@@ -45,16 +39,14 @@ To start a new project, all you need to do is make a directory with a cute name,
 cargo mobile init
 ```
 
-After some straightforward prompts, you'll be asked to select a template pack. Template packs are used to generate project boilerplate, i.e. using the `bevy` template pack gives you a minimal [Bevy](https://bevyengine.org/) project that runs out-of-the-box on desktop and mobile.
+After some straightforward prompts, you'll be asked to select a template pack. Template packs are used to generate project boilerplate, i.e. using the `wry` template pack gives you a [wry](https://github.com/tauri-apps/wry) project that runs out-of-the-box on desktop and mobile.
 
-| name      | info                                                                                                                              |
-| --------- | --------------------------------------------------------------------------------------------------------------------------------- |
-| bevy      | Minimal Bevy project derived from [sprite](https://github.com/bevyengine/bevy/blob/master/examples/2d/sprite.rs) example          |
-| bevy-demo | Bevy [breakout](https://github.com/bevyengine/bevy/blob/master/examples/game/breakout.rs) demo                                    |
-| wgpu      | Minimal wgpu project derived from [hello-triangle](https://github.com/gfx-rs/wgpu-rs/tree/master/examples/hello-triangle) example |
-| winit     | Minimal winit project derived from [window](https://github.com/rust-windowing/winit/blob/master/examples/window.rs) example          |
+| name      | info                                  |
+| --------- | ------------------------------------- |
+| wry       | Minimal wry project |
+| tauri     | Minimal tauri project |
```

Comment:
Could we take this chance to remove this template?
Or you think it's still better to have one?

Reply:
I think there is still some people using that template, so lets keep it for now.